lib/lotus/utils/path_prefix.rb

```diff
@@ -82,7 +82,7 @@ def relative_join(strings, separator = @separator)
         prefix = @string.gsub(@separator, separator)
 
         self.class.new(
-          Utils::Kernel.Array([prefix, strings]).join(separator),
+          [prefix, strings].flatten.compact.join(separator),
```

Comment:
This now follows the same pattern as Kernal.Array (as previously) but without the offending part (uniq).